.github/workflows/snyk.yml

```
@@ -0,0 +1,38 @@
+on:
+  schedule:
+    - cron: '0 4 1 * *'
+  workflow_dispatch: {}
+  push:
+    branches: [main, develop]
+  pull_request:
+    branches: [main, develop]
+
+jobs:
+  snyk-scan:
+    if: secrets.SNYK_TOKEN != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)